src/pyasm/search/sql_dumper.py

```diff
@@ -349,6 +350,23 @@ def dump_tactic_inserts(my, path, mode='sql'):
         column_info = SearchType.get_column_info(my.search_type)
 
         for sobject in my.sobjects:
+
+            if relative_dir_column:
```

Comment:
yes, this is to support a file per view.  Every sobject could open up a new different file.  It needs to be in the loop.